Flepper.QueryBuilder/Operators/Alias/AliasOperator.cs

```diff
@@ -14,8 +15,17 @@ public AliasOperator(StringBuilder command, IDictionary<string, object> paramete
 
         public IAliasOperator As(string alias)
         {
-            Columns = Columns.Select(c => c.TableAlias != null ? c : new SqlColumn($"[{alias}].{c}")).ToArray();
+            var withoutFunctions = Columns
+                .Where(c => !c.GetType().IsSubclassOf(typeof(FunctionOperator)))
+                .Select(c => c.TableAlias != null ? c : new SqlColumn($"[{alias}].{c}")).ToArray();
+            
+            var withFunctions = Columns
+                .Where(c => c.GetType().IsSubclassOf(typeof(FunctionOperator)))
```

Comment:
Using Reflection here, will make this process slower and you are looping over all columns twice.
It would be better let the SqlColumn with the responsability of producing the right value.
You can try doing an override on ToString of FunctionOperator